frontend/src/component/executiveDashboard/componentsStat/FlagStats/FlagStats.tsx

```diff
@@ -83,7 +83,11 @@ export const FlagStats: React.FC<IFlagStatsProps> = ({
             </StyledRingContainer>
 
             <ConditionallyRender
-                condition={flagsPerUser !== undefined && flagsPerUser !== ''}
+                condition={
+                    flagsPerUser !== undefined &&
+                    flagsPerUser !== '' &&
+                    flagsPerUser !== 'NaN'
```

Comment:
Wait, are you saying that the value we store here the string "NaN"? Or is it the value `NaN`?

I would expect this comparison to be done using the [Number.isNan](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN) function, but I don't know what the value we get from the API is in this case? If it is the string, then we should fix this on the API level, right?

Reply:
The value is coming from a calculation parsed to a string. So yes the value that gets here is  'NaN' and because of the other string comparisons here, I added it here for consistency
```  <FlagStats
                        count={summary.total}
                        flagsPerUser={
                            showAllProjects
                                ? (summary.total / users.total).toFixed(2)
                                : ''
                        }
                    />```